browser/Makefile.am

```
@@ -471,6 +471,21 @@ define global_file
 		@$(NODE) node_modules/uglify-js/bin/uglifyjs $< --output $@)
 endef
 
+define run_eslint_check
+	@$(NODE) node_modules/eslint/bin/eslint.js \
+		--resolve-plugins-relative-to $(abs_builddir) \
+		--ignore-path $(srcdir)/.eslintignore \
+		--config .eslintrc \
+		--no-eslintrc $(1)
+endef
+
+define run_prettier_check
+	@$(NODE) node_modules/.bin/prettier \
+		--ignore-path $(srcdir)/browser/.prettierignore \
+		--ignore-path $(srcdir)/browser/.beforeprettier \
```

Should this be like this instead?
```suggestion
		--ignore-path $(srcdir)/.prettierignore \
		--ignore-path $(srcdir)/.beforeprettier \
```

---

Ok, I will read documentation if I find some parameter

---

Oh, yes, you are right